src/prebid.js

```diff
@@ -334,7 +335,13 @@ $$PREBID_GLOBAL$$.removeAdUnit = function (adUnitCode) {
 
 $$PREBID_GLOBAL$$.clearAuction = function() {
   auctionRunning = false;
-  syncCookies(config.getConfig('cookieSyncDelay'));
+  // Only automatically sync if the publisher has not chosen to "enableOverride"
+  let userSyncConfig = config.getConfig('userSync') || {};
```

Comment:
@jaiminpanchal27  `userSync` can be set with `config.setConfig({userSync: null});`  though I'm not sure why anyone would set it to `null` or another `falsey` value, unless maybe they thought it would turn it off.